### DISCUSSION TITLE
Make semaphore links work when logged out (+ include post details in previews)

### DISCUSSION THREAD

**Discussion by wlonkly** [NickColley/semaphore#42] [New features and ideas]
Originally this feature request was going to be just for post details in previews, and I'd written:

> If I link a semaphore.social post on Slack, I see:
>
> <img width="576" alt="image" src="https://user-images.githubusercontent.com/1545955/216838328-e6f7c88c-ee9f-464c-ac91-e7144f9b6cab.png">
>
> Elk.zone previews the linked post instead:
>
> <img width="607" alt="image" src="https://user-images.githubusercontent.com/1545955/216838403-4c7e05a1-bdb7-4f5a-8862-fdfc1c33b82a.png">
>
> Obviously the URL model is different, but for public posts it would be nice to not have to find the original link to a post vs. copying the semaphore.social link. (It would also help to promote semaphore.social!) 

but I realize that that would also require links to public posts to display the posts, which would expand the scope of this feature request quite a bit... but it _would_ be nice to be able to pass semaphore.social links around.

**Comment by NickColley** [comment]
You're right to make this happen you have to consider much wider changes. Unfortunately while it's clearly a great feature to have it's out of scope for this project, thank you for raising it though.
